examples/hackweek_demo/run_demo.py

```diff
@@ -54,9 +52,7 @@
 
 def get_ticker(data):
     """Given transaction data, return a subset of the dataframe"""
-    # Return reverse of methods to put most recent transactions at the top
-    out = data[["blockNumber", "input_method"]].set_index("blockNumber").iloc[::-1]
-    return out
+    return data[["input_method"]].iloc[::-1]
```

Comment:
does this warrant having a function, its a one-liner?  admittedly I don't know what it does really, is it actually specific to transaction data?

Reply:
I think a function here is fine, this encapsulates the "ticker" panel that happens to be relatively simple in logic. Regardless, seeing the caller call "get_ticker" before displaying it seems better to me than the one liner.

Reply:
also, we may want to have more complicated ticker behaviour in the future